### DISCUSSION TITLE
Disable logging

### DISCUSSION THREAD

**Discussion by kincsescsaba** [lightbend-labs/scala-logging#382] [Q&A]
Hello there,

Is there a way to completely disable logging with an SBT task or from other parts of the source code?

Cheers,
Csaba

**Comment by gabrieljones** [comment]
Add the `slf4j-nop` logging provider to the runtime classpath. Remove all other slf4j logging providers from the runtime classpath.

> slf4j-nop-2.0.6.jar
    Binding/provider for [NOP](http://www.slf4j.org/api/org/slf4j/helpers/NOPLogger.html), silently discarding all logging.
> https://www.slf4j.org/manual.html
